src/Chapter1.hs

```diff
@@ -641,7 +664,13 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit x
+    | abs(x) < 10 = abs x
```

Comment:
Note that functions and arguments are always separated by spaces in Haskell 🙂 

```suggestion
    | abs x < 10 = abs x
```

Also, you can see that `abs n` is reused several times in this function. This usually means that you can move it into the `where` case and reuse everywhere 👍🏼